src/functionalTest/java/uk/gov/hmcts/reform/draftstore/data/DataAgent.java

```diff
@@ -27,8 +27,8 @@ public void setupDocumentForUser(String userId, String type, String document) th
         jsonbObj.setValue(document);
 
         jdbcTemplate.update(
-            "INSERT INTO draft_document (user_id, document_type, service, document) "
-                + "VALUES (:userId, :type, 'cmc', :document)",
+            "INSERT INTO draft_document (user_id, document_type, service, document, created, updated) "
+                + "VALUES (:userId, :type, 'cmc', :document, now(), now())",
```

Comment:
just putting it out there, what we do in the claim store is always insert at UTC and then use in LocalDateTime which is easier to work with.

Reply:
We return timezone info (although it's always UTC) to follow https://hmcts.github.io/restful-api-standards/#126